x-pack/solutions/chat/plugins/workchat-app/common/http_api/agents.ts

```diff
@@ -20,3 +20,5 @@ export type CreateAgentResponse = { success: false } | { success: true; agent: A
 export type UpdateAgentPayload = Omit<AgentCreateRequest, 'id'>;
 
 export type UpdateAgentResponse = { success: false } | { success: true; agent: Agent };
+
+export type DeleteAgentResponse = { success: false } | { success: true; agent: Agent };
```

Comment:
nit: convention is to just return `{ success: true }` when deletnig, no need to return deleted agent in response ?

Reply:
done